src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -50,6 +78,8 @@ public function isCandidate(Tokens $tokens)
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
+        $callback = $this->fixCallback;
```

Comment:
I'm having a little issue with the naming of all this. It's a callable yes, but it is in no way a _callback_. It's (just) the fix function.

Reply:
Ok, will make something meaningful.

Reply:
It's still not a `$callback` 😉

Reply:
I renamed it to `$fixFunction`, do you suggest something else ?

Reply:
You renamed the property but not the variable, was the point I unsuccessfully was trying to make.

Reply:
oops, fixed now.